packages/core/src/DreifussWysiwygEditorDemo.tsx

```diff
@@ -74,20 +74,43 @@ const value: any = [
   //   ]
   // },
   // {type: 'paragraph', children: [{text: 'Emojis: 😄'}]}
+
+  {
+    type: 'paragraph',
+    children: [{type: 'link', url: 'http://google.com', children: [{text: 'Links: Add links.'}]}]
+  },
+  {
+    type: 'paragraph',
+    children: [
+      {text: 'Bold: ', bold: true},
+      // {bold: true, text: 'Make', color: '#c21414'},
+      {bold: true, text: ' the selected text bold.'}
+    ]
+  }
 ]
 
 const DreifussWysiwygEditorDemo = () => (
-  <div style={{minHeight: 400, padding: 30}}>
-    <h1>RichText Component Demo</h1>
-    <DreifussWysiwygEditor
-      // charactersCount={count => {
-      //   console.log(count)
-      // }}
-      onChange={(data: any) => {
-        // console.log(JSON.stringify(data))
-      }}
-      initialValue={value}
-    />
+  <div style={{display: 'flex'}}>
+    <div style={{minHeight: 400, padding: 30}}>
+      <h1>RichText Component Demo</h1>
+      <DreifussWysiwygEditor
+        onChange={e => {
+          console.log(e)
```

Comment:
Do you need the console.log ?

Reply:
@camilleory it won't affect the package as it's in the Demo component 👍